python/server/ServerWrapper.cpp

```diff
@@ -321,6 +321,20 @@ namespace {
         return py_items;
     }
 
+    // Wrapper for starting buildings
+    list LoadStartingBuildings() {
```

Comment:
Yes.

Moving it out of the python into C++ made it faster on its own.  

I said that parsing them in a central location and early would be optimal, but this PR just moves them to a central location and a standard syntax.

Collecting all the startup parses and doing them early was on the list for this branch later, but I ran out of steam.